modules/ROOT/pages/clustering/servers.adoc

```diff
@@ -31,11 +31,50 @@ These free servers are not yet part of the cluster and cannot be allocated to ho
 When first discovered, a server's name defaults to the value of its generated server ID.
 ====
 
+[[server-enabled-state]]
 === Enabled state
 
 A server in the free state needs to be explicitly enabled in order to be considered an active member of the cluster.
 The command `ENABLE SERVER server name` is used to transition a server into this _Enabled_ state.
-Subsequently, the server may be allocated databases to host.
+The server's initial name is its ID.
+If the server is already enabled and the command is executed with the same options, nothing is changed.
+
+The possible options when enabling a server are:
+
+[options="header", width="100%", cols="2a,2,^.^"]
+|===
+| Option
+| Allowed values
+| Description
+
+| modeConstraint
+| `PRIMARY`, `SECONDARY`, `NONE`
+| Databases may only be hosted on the server in the mode specified by the constraint.
+`None` means there is no constraint and any mode is allowed.
+
+| allowedDatabases
+| list of database names, e.g. `["db1", "db2"]`
+| Only databases matching the specified names may be hosted on the server.
+This may not be specified in combination with `deniedDatabases`.
+
+| deniedDatabases
+| list of database names, e.g. `["db1", "db2"]`
+| Only databases **not** matching the specified names may be hosted on the server.
+This may not be specified in combination with `allowedDatabases`.
+
+| tags
+| list of server tags, e.g. `["tag1", "tag2"]`
+| List of server tags used during database allocation and for load balancing and routing policies.
+label:new[Introduced in 5.6]
+|===
+
+[NOTE]
+====
+When a server is enabled, if `tags` are not provided in `OPTIONS`, the default server tags are taken from the setting `initial.server.tags`.
```

Comment:
This is valid for the other options as well. The default values are coming from `initial.server.mode_constraint`, `initial.server.allowed_databases`, `initial.server.denied_databases`

Reply:
Added!